gst/nnstreamer/tensor_transform/tensor_transform.c

```diff
@@ -765,7 +765,11 @@ gst_tensor_transform_set_option_data (GstTensorTransform * filter)
     case GTT_TYPECAST:
     {
       if (g_regex_match_simple (REGEX_TYPECAST_OPTION, filter->option, 0, 0)) {
-        filter->data_typecast.to = gst_tensor_get_type (filter->option);
+        gint type_to = gst_tensor_get_type (filter->option);
+        if (type_to == _NNS_INT64 || type_to == _NNS_UINT64) {
```

Comment:
requires definition HAVE_ORC for filter->orc_supported

#ifdef HAVE_ORC
... condition to set filter->orc_supported
#endif

Reply:
Thanks! Updated.